components/nifosg/particle.cpp

```diff
@@ -191,6 +191,7 @@ void ParticleColorAffector::operate(osgParticle::Particle* particle, double /* d
 
     particle->setColorRange(osgParticle::rangev4(color, color));
     particle->setAlphaRange(osgParticle::rangef(alpha, alpha));
+
```

Comment:
Please discard this redundant change and squash commits.

Reply:
> 
> 
> reopened just for a win32 test
I'll clean that pr as test is pretty satisfying..don't know why i closed it

Reply:
ah yes i remember, someone nags me about stuff that wasn't in nif